server/tools/docker-entrypoint.sh

```diff
@@ -51,9 +51,9 @@ SYS_PROPS+=" $BIND_OPTS"
 # Configuration #
 #################
 
-# If the "-c" parameter is not present, append the HA profile.
-if echo "$@" | egrep -v -- "-c "; then
-    SYS_PROPS+=" -c standalone-ha.xml"
+# If the server configuration parameter is not present, append the HA profile.
+if echo "$@" | egrep -v -- '-c |-c=|--server-config |--server-config='; then
```

Comment:
To be safer, one could actually add a " " before every "-" so it is clear that it is a configuration switch. But maybe it is also fine without...